osdep/main-fn-win.c

```diff
@@ -9,6 +9,18 @@
 #include "osdep/terminal.h"
 #include "osdep/main-fn.h"
 
+#ifndef HEAP_OPTIMIZE_RESOURCES_CURRENT_VERSION
+
+#define HeapOptimizeResources 3
```

Comment:
`enum { HeapEnableTerminationOnCorruption = 3 }` might be better for consistency

Reply:
done